### PR TITLE
Fix static libraries being treated as dynamic libs

### PR DESCRIPTION
**Pull request by Squareys**
Hi @mosra !

I applied the patch you sent me, and copied the changed to OvrIntegration. Works like a charm :wink: 

Greetings, Squareys.

### CODE REVIEW

File: src/Magnum/OvrIntegration/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +26,14 @@
...
 find_package(OVR REQUIRED)
 
+if(BUILD_STATIC)
+    set(MAGNUM_OVRINTEGRATION_BUILD_STATIC 1)
+endif()
+
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
+               ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
+
+
```
Comment (mosra) [inline src/Magnum/OvrIntegration/CMakeLists.txt]: Superfluous newline ;)

Comment (Squareys) [inline src/Magnum/OvrIntegration/CMakeLists.txt]: Done ;)

Comment (Squareys) [inline src/Magnum/OvrIntegration/CMakeLists.txt]: *err*

Comment (Squareys) [inline src/Magnum/OvrIntegration/CMakeLists.txt]: Now, done. Messed up my git commands :)

**Comment by mosra** [comment]
:+1:

**Comment by Squareys** [comment]
Thanks! :D

**Pull request merged**